luminoth/eval.py

```diff
@@ -53,19 +52,20 @@ def eval(dataset_split, config_files, watch, from_global_step,
     # Disable data augmentation.
     config.dataset.data_augmentation = []
 
+    # Override max detections with specified value.
+    config.model.rcnn.proposals.total_max_detections = max_detections
```

Comment:
Does this break support por RPN only? Maybe add `post_nms_top_n` (renaming it while at it) support?

Reply:
Noted.